model/concourse.py

```diff
@@ -385,6 +386,12 @@ def secret_name_from_team(team_name: str, key_generation: int) -> str:
         return f'{cfg_name_from_team(team_name)}-{key_generation}'
 
 
+@dataclasses.dataclass(frozen=True)
+class TrustedTeams:
+    github_hostname: str
+    teams: list[str]
```

Comment:
list is mutable. declaring the dataclass as frozen is inconsistent w/ that. suggestion: declare the class as mutable (non-frozen)